### PR TITLE
Add "tableOfContents" with unique id (#1237)

### PR DESCRIPTION
**Pull request by TobiasNx**
See #1237

### CODE REVIEW

File: src/main/resources/alma/common/fields.xml [hunk 1]
```diff
@@ -489,6 +489,23 @@
 		</entity>
 	-->
 
+    <entity name="tableOfContents[]" flushWith="record">
+    <entity name="" sameEntity="true" reset="true">
+      <if>
+        <all flushWith="8564?" sameEntity="true" reset="true">
+          <data source="8564?.3">
+				    <regexp match="^[Ii][Nn][Hh][aA][lL][tT][sS][vV]"/>
+          </data>
```
Comment (dr0i) [inline src/main/resources/alma/common/fields.xml:498]: formatting seems broken

**Review by dr0i — CHANGES_REQUESTED** [review]
otherwise +1

**Comment by dr0i** [comment]
Also, if you like, try to solve conflicts.

File: src/main/resources/alma/common/fields.xml (hunk 1 above)
Comment (TobiasNx) [reply src/main/resources/alma/common/fields.xml:498]: done

**Comment by dr0i** [comment]
Ignore the fails checks - a play resource cannot be accessed atm.

**Comment by dr0i** [comment]
See https://alma.lobid.org/resources/search?q=HT000161712&format=json. Closed.

**Pull request merged**